backend/src/Manager/SupportManager.php

```diff
@@ -33,14 +33,14 @@ public function createSupport(SupportCreateRequest $request, $uuid)
         return $entity;
     }
 
-    public function getReports()
+    public function getSupports()
     {
-        return $this->supportEntityRepository->getReports();
+        return $this->supportEntityRepository->getSupports();
     }
 
-    public function getReport($id)
+    public function getSupport($id)
     {
-        return $this->supportEntityRepository->getReport($id);
+        return $this->supportEntityRepository->getSupport($id);
     }
 
     public function getReportByUuid($uuid)
```

Comment:
this need to be more readable uuid is not good

Reply:
ok